src/components/common/bullets-colin-link.vue

```diff
@@ -0,0 +1,165 @@
+          class="mb-0 pb-4"
+          label="Named Company"
+          :value="CompanyType.NAMED_COMPANY"
+        />
+        <v-radio id="numbered-company-radio"
+          label="Numbered Company"
+          :value="CompanyType.NUMBERED_COMPANY"
+        />
+      </v-radio-group>
+    </v-col>
+    <v-col cols="12" sm="9">
+      <div v-if="selectedCompanyType === CompanyType.NAMED_COMPANY">
+        <v-row>
+          <slot name="name-input-slot">Name Input</slot>
+          <template v-if="showDesignation">
+            <slot name="designation">Designation</slot>
+          </template>
+        </v-row>
+      </div>
+      <div v-else>
+        <ul class="bullet-points">
+          <li v-for="bulletPoint in bulletPoints" :key="bulletPoint">
+            {{ bulletPoint }}
+          </li>
+        </ul>
+        <div class="btn-spacing" v-if="colinButton">
+          <v-btn class="px-9" :href="colinLink" target="_blank">
+            Go to Corporate Online to Register <v-icon small class="ml-1">mdi-open-in-new</v-icon>
+          </v-btn>
+        </div>
+        <div class="btn-spacing" v-else>
+          <v-btn
+            class="px-9"
+            @click="incorporateNowClicked()"
+          >
+            Incorporate Now
+          </v-btn>
+        </div>
+      </div>
+    </v-col>
+  </v-row>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Mixins, Prop } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { CompanyType, EntityType } from '@/enums'
+import NameInput from '@/components/new-request/name-input.vue'
+import { Navigate } from '@/plugins'
+import { NrAffiliationMixin } from '@/mixins'
+
+@Component({
+  components: {
+    NameInput
+  }
+})
+export default class BulletsColinLink extends Mixins(NrAffiliationMixin) {
+  /** The selected business type. */
+  @Prop({ default: '' }) readonly businessType!: EntityType
+
+  /** Show Colin Button. */
+  @Prop({ default: false }) readonly colinButton!: boolean
+
+  /** Show Designation Dropdown. */
+  @Prop({ default: false }) readonly showDesignation!: boolean
+
+  /** Whether user is authenticated. */
+  @Getter getIsAuthenticated!: boolean
+
+  // Local properties
+  selectedCompanyType: CompanyType = null
+  bulletPoints = [
+    'Your business name will be the Incorporation Number assinged by the Registry.',
+    'You can change your business name at a later date.',
+    'It is not possible to request a specific Incorporation Number.'
+  ]
+  readonly colinLink = sessionStorage.getItem('CORPORATE_ONLINE_URL')
+
+  // For template
+  readonly CompanyType = CompanyType
+
+  /**
+   * The alternate codes for entity types.
+   * Alternate codes are used in Entities UIs.
+   */
+  entityTypeAlternateCode (entityType: string): string {
+    switch (entityType) {
+      case EntityType.BC: return 'BEN'
+      case EntityType.CR: return 'BC'
+      case EntityType.UL: return 'ULC'
+      default: return entityType
```

Comment:
Since CorpTypeCd is now imported in this app, you should use that enum instead of strings.

You can also use this type in incorporateNow() and createBusinessIA().

Reply:
Please let me know if we can resolve this conversation Sev. Ditto the one below it.

Reply:
Yup, code looks good. Resolving.